src/gantt.ts

```diff
@@ -589,7 +589,7 @@ module powerbi.extensibility.visual {
                             identity: selectionId
                         };
 
-                        task.end = d3.time.day.offset(task.start, task.duration);
```

Comment:
ok @MrMeison, Any option for having this change anyway? Like introducing a legacy button in the "Gantt Date type" menu that are "On" by default?
If doing so, I can also add "Hours", "Minutes", and "Seconds" as additional type of scale
I realy think a Gantt chart is to important to leave in current condition